docs/docsite/rst/porting_guides/porting_guide_2.10.rst

```diff
@@ -115,6 +115,9 @@ Noteworthy plugin changes
 -------------------------
 
 * The ``hashi_vault`` lookup plugin now returns the latest version when using the KV v2 secrets engine. Previously, it returned all versions of the secret which required additional steps to extract and filter the desired version.
+* The ``setup`` module now provides ``ansible_processor_nproc`` which
+  reflects the number of vcpus available to processes (falls back to the
+  number of vcpus available to the scheduler)
```

Comment:
This isn't a plugin change (fact gathering is really an internal implementation). Since this is a change to the default gathered facts, put this under the `Playbook` section. It can just be its own paragraph.

I would omit the mention of ``setup`` and just describe the new fact.
```suggestion
A new fact, ``ansible_processor_nproc``, reflects the number of vcpus available. This is useful in containers or other environments where limits are applied and the CPUs available on the host are not the same as CPUs available to the container. 
```

Reply:
Proposal pushed.